### ISSUE TITLE
Mobile compatibility

### ISSUE THREAD

**Issue by nickatomlin** [nickatomlin/Parks#16]
When viewing the site on mobile (browser: iOS 9.3/10 Safari), the input text field does not appear. We also notice #15 on the mobile Safari web browser.

**Comment by nickatomlin** [comment]
Issue #15 is no longer a problem on mobile as of 22e4e27b1fae0f25348da7a1c185a01a5eb61a1c. The input text field is still missing and similarly does not appear in the original Hack Spots version.

**Comment by nickatomlin** [comment]
Fixed via af333be9014c2685367846764fdbf06b69b5e241 by editing the following snippet of code in site.css:

```
@media only screen and (-webkit-min-device-pixel-ratio: 2) and (min-width: 320px) and (max-width: 1024px) {
  body {margin: 0px; }
  .header {background-image: none;}
  #map {width: 90%; max-width: 100%;}
  #rightSide {width: 100%; max-width: 100%;}
  .resetMap, .clear, .filterText {display: none;}
  #wrapper {max-width: 460px; padding-top: 20px;}
  #hackSpotsTable, #map {min-width: 50%;}
}
```

Notably, we can change the list of classes/IDs appearing before `{display: none}` to hide them on mobile. Note also that the background image in the header does not appear on mobile.
